### DISCUSSION TITLE
What's the state of the project?

### DISCUSSION THREAD

**Discussion by uduse** [guildai/guildai#523] [Q&A] [answered]
Hi, I have been a long-time guildai fan, but I have been hesitant to adopt in my new projects because it seems the project is abandoned. Does anyone has some insights to share?

**Comment by gar1t** [comment, marked as answer]
A few big changes have taken place over the past year or so. 

- Posit acquired Guild AI in 2021 and funded its ongoing development through two major releases - we're extremely grateful for their support
- Last year, Posit made a decision to focus on their core business and the Guild team (myself and @alxmke) left to focus on experiment tracking tools - at the the time it wasn't clear if we were going to fork Guild or start afresh
- We ended up starting a new open source project - Gage - which is under development at https://github.com/gageml/gage
- Gage is a ground-up rewrite of Guild; it's not backward compatible in any way; it's not yet released but is quite functional and of similar quality to Guild

Unfortunately there are no docs for Gage currently. If you're interested in taking a closer look, I'd be very happy to get you started and address any issues you have as a top priority. An official release with docs are coming soon (targeting the end of April!)

What does this mean for Guild? If there's a straight forward fix for an issue, we'll fix it. Our main focus though is on Gage. The good news here is that Gage is poised to overtake Guild. I'm really excited about its current state and trajectory.

We'll have more details as systems come online.

In the meantime, please feel free to use this thread to get more info. If there are pressing issues with Guild, please open a new issue, or point me to an existing issue you need addressed.

**Reply by uduse** [reply]
That's really good to know. Thanks for the detailed response and I am excited to give gage a try in the future!
